ush/soca/prep_marine_obs.py

```diff
@@ -63,29 +62,38 @@
 }
 
 
-def obs_fetch(obsprepSpace):
+def obs_fetch(obsprepSpace, cycles):
 
     subDir = obsprepSpace['dmpdir subdir']
     filepattern = obsprepSpace['dmpdir regex']
-
-    dataDir = os.path.join(cycDir, subDir)
-    # TODO: check the existence of this
-    print('dataDir:', dataDir)
     matchingFiles = []
+    fileCopy = []
+    targetFiles = []
+
+    for cycle in cycles:
+
+        cycleDate = cycle.strftime('%Y%m%d')
+        cycleHour = cycle.strftime('%H')
+
+        dataDir = os.path.join(DMPDIR, RUN + '.' + cycleDate, cycleHour, subDir)
+
+        # TODO: check the existence of this
+        print('dataDir:', dataDir)
 
-    for root, _, files in os.walk(dataDir):
-        for filename in fnmatch.filter(files, filepattern):
-            matchingFiles.append(filename)
+        for root, _, files in os.walk(dataDir):
+            for filename in fnmatch.filter(files, filepattern):
+                targetFile = cycleDate + cycleHour + '-' + filename
+                matchingFiles.append((dataDir, filename, targetFile))
 
-    obsCopy = []
-    for obsSource in matchingFiles:
-        obsPath = os.path.join(dataDir, obsSource)
-        obsDestination = os.path.join(COMIN_OBS, obsSource)
-        obsCopy.append([obsPath, obsDestination])
+    for matchingFile in matchingFiles:
+        filePath = os.path.join(matchingFile[0], matchingFile[1])
+        fileDestination = os.path.join(COMIN_OBS, matchingFile[2])
+        fileCopy.append([filePath, fileDestination])
 
-    print(f"obsCopy: {obsCopy}")
+    print(f"fileCopy: {fileCopy}")
```

Comment:
since logging is working now ...

Reply:
no need to retest, and since you're on a roll @AndrewEichmann-NOAA , could you swap that to logging pretty please?

Reply:
@guillaumevernieres done

Reply:
@guillaumevernieres done and short-tested